### ISSUE TITLE
500 Error on Ubuntu 14.04

### ISSUE THREAD

**Issue by ahallicks** [pa11y/pa11y-dashboard#89]
Hey guys,

Love the tool! It works great via CLI but I cannot get the dashboard to work at all. I am running:

Ubuntu 14.04
Node v0.10.25
And the latest pa11y-dashboard version 1.6.0

I have run through the setup instructions and everything _seemed_ to install just fine. However, when navigating to the created url:4000 I get the following:

```
Error: Error 404
    at Request._callback (/path-to-pa11y-dashboard/node_modules/pa11y-webservice-client-node/lib/client.js:124:16)
    at Request.self.callback (/path-to-pa11y-dashboard/node_modules/request/request.js:129:22)
    at Request.EventEmitter.emit (events.js:98:17)
    at Request.<anonymous> (/path-to-pa11y-dashboard/node_modules/request/request.js:873:14)
    at Request.EventEmitter.emit (events.js:117:20)
    at IncomingMessage.<anonymous> (/path-to-pa11y-dashboard/node_modules/request/request.js:824:12)
    at IncomingMessage.EventEmitter.emit (events.js:117:20)
    at _stream_readable.js:920:16
    at process._tickDomainCallback (node.js:459:13)
```

This logs in both the test output, which is styled nicely, and the console.

Any help would be appreciated.

**Comment by josebolos** [comment]
Hi @ahallicks,

I've just realised you posted this problem long ago, did you ever managed to fix it?

I was just having a look at this but I haven't been able to replicate the problem that you're seeing. The steps that I've followed are:
- Create a Ubuntu 14.04 VM:
  `vagrant init ubuntu/trusty64; vagrant up --provider virtualbox`
- SSH into the VM:
  `vagrant ssh`
- Install some needed packages:
  `sudo apt -V install nodejs npm git mongodb`
- Install pa11y-webservice as per [its instructions](https://github.com/springernature/pa11y-webservice/blob/master/README.md). I think it's not very clear from the README but pa11y-dashboard needs pa11y-webservice running:

```
git clone https://github.com/springernature/pa11y-webservice.git
cd pa11y-webservice
cp config/development.sample.json config/development.json
cp config/production.sample.json config/production.json
cp config/test.sample.json config/test.json
```
- Run pa11y-webservice in test mode, in the background:
  `NODE_ENV=test node index.js &`
- Install pa11y-dashboard:

```
git clone https://github.com/springernature/pa11y-dashboard.git
cd pa11y-dashboard
cp config/development.sample.json config/development.json
cp config/production.sample.json config/production.json
cp config/test.sample.json config/test.json
```
- Run pa11y-dashboard in test mode, in the background:
  `NODE_ENV=test node index.js &`
- Open the page (I used `curl`):
  `curl -i "http://localhost:4000`

`curl` returns a 200 OK

I didn't test any further than this, but it looks like this is the point where you where getting a 500 error.
